l10n_us_gaap/__manifest__.py

```diff
@@ -0,0 +1,20 @@
+{
+    'name': 'United States Sample GAAP Chart of Accounts',
+    'version': '12.0.1.0.0',
+    'development_status': 'Mature',
```

Comment:
@dduarte-odoogap We need to add unit tests to reach `Stable`:
https://odoo-community.org/page/module-maturity-levels